From d4425af9646964c684a31f73e0d3dcb3a2b46dd9 Mon Sep 17 00:00:00 2001
From: "dongwan0605.kim" <dongwan0605.kim@lge.com>
Date: Sun, 6 May 2018 14:57:00 +0900
Subject: [PATCH 05/21] Bind app name to menu row when notification updated

Fix descriptions of notification menu items was reset after notification updated.
When notification updated, menu row will be reinflated.
So we need to bind app name again.

Test: Talkback ON > Update existing notification > Swipe notification
      > Touch snooze or gear button and check content descriptions

Change-Id: I070712e42f9405b11446b20ac030d3e2cefa1fa3
---
 .../android/systemui/statusbar/ExpandableNotificationRow.java    | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/ExpandableNotificationRow.java b/packages/SystemUI/src/com/android/systemui/statusbar/ExpandableNotificationRow.java
index 966e7899724..b24838fefee 100644
--- a/packages/SystemUI/src/com/android/systemui/statusbar/ExpandableNotificationRow.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/ExpandableNotificationRow.java
@@ -360,6 +360,7 @@ public class ExpandableNotificationRow extends ActivatableNotificationView
         updateNotificationColor();
         if (mMenuRow != null) {
             mMenuRow.onNotificationUpdated(mStatusBarNotification);
+            mMenuRow.setAppName(mAppName);
         }
         if (mIsSummaryWithChildren) {
             mChildrenContainer.recreateNotificationHeader(mExpandClickListener);
-- 
2.17.1

